packages/typespec-client-generator-core/src/license.ts

```diff
@@ -0,0 +1,139 @@
+import { LicenseInfo, TCGCContext } from "./interfaces.js";
+
+export const licenseMap: { [key: string]: LicenseInfo } = {
+  "MIT License": {
+    name: "MIT License",
+    link: "https://mit-license.org",
+    company: "",
+    description: `Copyright © <company>\n
+\n
+Permission is hereby granted, free of charge, to any person obtaining a copy\n
+of this software and associated documentation files (the “Software”), to deal\n
+in the Software without restriction, including without limitation the rights\n
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell\n
+copies of the Software, and to permit persons to whom the Software is\n
+furnished to do so, subject to the following conditions:\n
+\n
+The above copyright notice and this permission notice shall be included in\n
+all copies or substantial portions of the Software.\n
+\n
+THE SOFTWARE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR\n
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,\n
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE\n
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER\n
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,\n
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN\n
+THE SOFTWARE.\n`,
```

Comment:
Does any SDK use the full form?

Java uses a short form
```
Copyright (c) Microsoft Corporation. All rights reserved.
Licensed under the MIT License.
```

Reply:
python use the full form in `LICENSE` file. do you think i need to add a short form into the type? or let emitter to do it with `license.name` and `license.company`?

Reply:
added `header`.